### PR TITLE
[FIX] Detox: wallet-tests.spec.js

### PR DESCRIPTION
**Pull request by chrisleewilcox**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**
Fixed `wallet-tests.spec.js`
- refactored NFT steps
- removed steps for validating drawer view that doesn't happen anymore with UI changes
- added ERC-1155 token info to `Accounts.js`

**Screenshots/Recordings**
image

**Issue**
ERC-1155 token data needs to be updated for the bitrise SRP wallet that owns an ERC-1155 token.  This can be easily done now by updating `Account.js` ERC-1155 token data.

The token data that existed prior to these changes was failing during test with error indicating account is not the owner.

Progresses #???

**Checklist**

* [x] There is a related GitHub issue
* [x] Tests are included if applicable
* [x] Any added code is fully documented

### CODE REVIEW

File: wdio/helpers/Accounts.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ const CORRECT_PASSWORD = `12345678`;
 const SHORT_PASSWORD = `1234567`;
 
 const INCORRECT_PASSWORD = `12345679`;
+const ERC_1155_TOKEN = `0xCBBA076Cd4c35432203F71fc5CaeEBA38C25aa2C`;
```
Comment (cortisiko) [inline wdio/helpers/Accounts.js:8]: I don't think having token data mixed with account data is a good idea.  you should probably create a json file that holds token data then in the test read the token data from the json.

**Review by cortisiko — CHANGES_REQUESTED** [review]
The code looks good, just one comment.

File: wdio/helpers/Accounts.js (hunk 1 above)
Comment (chrisleewilcox) [reply wdio/helpers/Accounts.js]: Updated.  Moved collectible data to collectible.json and removed code from Accounts.js.

Comment (cortisiko) [reply wdio/helpers/Accounts.js]: Nice, looks good!

**Review by cortisiko — APPROVED** [review]
🌮 🌮 🌮

**Comment by cortisiko** [comment]
update ticket description to reflect where the 1155 token data is. Once that is done, feel free to merge

**Pull request merged**